fake.go

```diff
@@ -519,6 +521,177 @@ func (fake *Fake) Dump() string {
 	return buf.String()
 }
 
+// extractComment removes comment section from a given string and returns it together with comment string.
+// If a comment wasn't found, an empty comment string will be returned.
```

Comment:
`comment ""` is different from no comment. You need to return a `*string` for the comment so you can return `nil` if there wasn't one